apps/omg_watcher_info/test/omg_watcher_info/order_fee_fetcher_test.exs

```diff
@@ -0,0 +1,96 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.OrderFeeFetcherTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.WatcherInfo.Fixtures
+
+  alias OMG.Eth
+  alias OMG.Utils.HttpRPC.Encoding
+  alias OMG.WatcherInfo.OrderFeeFetcher
+  alias OMG.WatcherInfo.TestServer
+  alias OMG.WireFormatTypes
+
+  @eth Eth.zero_address()
+  @not_eth <<1::160>>
+  @tx_type WireFormatTypes.tx_type_for(:tx_payment_v1)
+  @str_tx_type Integer.to_string(@tx_type)
+
+  describe "add_fee_to_order/2" do
+    @tag fixtures: [:test_server]
```

Comment:
Hoping to move away from fixtures. There aren't that many tests using `:test_server` but this PR adds quite a lot. Could we convert this into a helper function instead?